test/benchmarks/test_program.py

```diff
@@ -0,0 +1,38 @@
+from typing import Callable
+
+import pytest
+
+from pyquil.quil import Program
+
+
+@pytest.fixture
+def over_9000_program() -> Program:
+    with open("test/benchmarks/fixtures/over-9000.quil") as f:
+        return Program(f.read())
+
+@pytest.fixture
+def large_program() -> Program:
+    with open("test/benchmarks/fixtures/large_with_calibrations.quil") as f:
+        return Program(f.read())
```

Comment:
Maybe worth making a parameterized fixture that operates over multiple programs? That way we can e.g. `test_instructions` for programs of varying sizes or `test_copy_everything_except_instructions` for programs with differing numbers of instructions, calibrations, etc.